src/shared/com/example/model/account.cljc

```diff
@@ -34,6 +34,7 @@
    ao/required?                                                    true
    ao/schema                                                       :production
    :com.fulcrologic.rad.database-adapters.datomic/attribute-schema {:db/unique :db.unique/value}
+   :com.fulcrologic.rad.database-adapters.datomic-cloud/attribute-schema {:db/unique :db.unique/value}
```

Comment:
We should not create new attributes that represent the same exact data. Is there any diff in datomic-cloud schema?  datomic-free?   datomic-peer?   datomic-client? As far as I know it's all the same schema.